### PR TITLE
remove aks engine instructions from README

### LINKED ISSUE #136: Update README.md to remove references to aks-engine
[aks-engine](https://github.com/Azure/aks-engine#project-status) is deprecated. 

- We should restructure our readme to reference general instructions for setup, rotation instead of the aks-engine cluster config example.
- Add reference to AKS docs when https://github.com/Azure/AKS/issues/845 is complete.

### PR DESCRIPTION
**Pull request by salaxander**
**Reason for Change**:
AKS engine is being deprecated, this is a docs change to remove the reference to it from the README.

**Issue Fixed**:
fixes #136

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,6 @@ From the Kubernetes documentation on [Encrypting Secret Data at Rest]:
...
 
-### 🎁 aks-engine
-
-[AKS Engine] creates customized Kubernetes clusters on Azure.
-
-Follow the AKS Engine documentation about [Azure Key Vault Data Encryption] and refer to the [example cluster configuration] to create a Kubernetes cluster with KMS Plugin for Key Vault automatically configured. Once the cluster is running, there will be an Azure Key Vault containing a new key in the same resource group as the cluster.
-
 ### Azure Kubernetes Service (AKS)
 
 Azure Kubernetes Service ([AKS]) creates managed, supported Kubernetes clusters on Azure.
```
Comment (nilekhc) [inline README.md:33]: I think we should also point to AKS-KMS preview doc in this section for users to try it out.
https://docs.microsoft.com/en-us/azure/aks/use-kms-etcd-encryption

Currently links in this section points to general container security

Comment (salaxander) [reply README.md:33]: Updated!

Comment (nilekhc) [reply README.md:33]: Thanks @salaxander. One more small nit. There are also references of AKS-Engine in manual installation doc. We should remove that too.

https://github.com/Azure/kubernetes-kms/blob/master/docs/manual-install.md#obtaining-the-id-of-the-cluster-managed-identityservice-principal

Comment (salaxander) [reply README.md:33]: One more update pushed, thanks for the review!

File: README.md [hunk 2]
```diff
@@ -28,19 +28,13 @@ From the Kubernetes documentation on [Encrypting Secret Data at Rest]:
...
 
-For more information about K8s secrets in AKS follow this [doc](https://docs.microsoft.com/en-us/azure/aks/concepts-security#kubernetes-secrets).
+To try out the KMS plugin for KeyValut on AKS, follow this [doc](https://docs.microsoft.com/en-us/azure/aks/use-kms-etcd-encryption).
```
Comment (aramase) [inline README.md:35]:
```suggestion
To try out the KMS plugin for Key Vault on AKS, follow this [doc](https://docs.microsoft.com/en-us/azure/aks/use-kms-etcd-encryption).
```

**Review by aramase — COMMENTED** [review]
Thanks for the PR!

Just one small nit.

**Pull request merged**